packages/firestore/src/remote/connection.ts

```diff
@@ -113,6 +113,7 @@ export interface Connection {
  * be called if the stream successfully established a connection.
```

Comment:
Can you clarify the difference between onOpen and onConnected in the documentation? Currently the docs state that "onOpen will only be called if the stream successfully established a connection." That definition of onOpen is what I would expect for onConneted.

Reply:
I've updated the docs. LMK if they clarify.

The naming "onOpen" is somewhat unfortunate, as it does not actually indicate that an opened connection with the backend has been established. I think at one point this was the meaning, but the channel is now optimized to send the "handshake" with the first message to avoid an extra round-trip with the backend. So WebChannelConnection now sends onOpen as soon as it is ready to send a message so that it can send the handshake along with the first message. I added onConnected to send an event when an _actual_ connection with the backend has been established.